feature/device/device_test.go

```diff
@@ -113,10 +111,6 @@ func TestFullReplaceRequest(t *testing.T) {
 		name: "device with basic LLDP and BGP",
```

Comment:
Remove LLDP from desc?

Reply:
sigh. I will fix in the next PR. Every time I push a change, the LGTMs are removed.